.github/workflows/release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       matrix:
         target:
         - x86_64-unknown-linux-gnu
-        - aarch64-unknown-linux-gnu
+#        - aarch64-unknown-linux-gnu
```

Comment:
Should this changes be reverted?

Reply:
I hope so, last time it was a problem with CI, that's why it commented.